prow/plugins/project/project.go

```diff
@@ -315,6 +307,19 @@ func handle(gc githubClient, log *logrus.Entry, e *github.GenericCommentEvent, p
 		return nil
 	}
 
+	// Clear issue/PR from project if command is to clear
+	if shouldClear && (existingProjectCard != nil) {
+		if err := gc.DeleteProjectCard(existingProjectCard.ID); err != nil {
+			return err
+		}
+		msg = fmt.Sprintf(successClearingProjectMsg, proposedProject)
+		return gc.CreateComment(org, repo, e.Number, plugins.FormatResponseRaw(e.Body, e.HTMLURL, e.User.Login, msg))
+	} else {
+		msg = fmt.Sprintf(failedClearingProjectMsg, proposedProject, e.HTMLURL)
+		return gc.CreateComment(org, repo, e.Number, plugins.FormatResponseRaw(e.Body, e.HTMLURL, e.User.Login, msg))
+	}
+
```

Comment:
Everything from this point on is dead code now since both outcomes of the new condition result in a `return`. (Consider the `shouldClear == false` case.)
We should probably have something like the following instead:
```go
if shouldClear {
  if existingProjectCard != nil {
    // successful clear case...
    return
  }
  // failed to clear case...
  return
}
// shouldClear == false case...
```